crd/nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -68,6 +68,14 @@ type NodeNetworkConfigStatus struct {
 	NetworkContainers []NetworkContainer `json:"networkContainers,omitempty"`
 }
 
+// NetworkContainerStatus defines the status of the Network Container
+type NetworkContainerStatus struct {
+	Timestamp metav1.Time `json:"timestamp,omitempty"`
+	Status    string      `json:"status,omitempty"`
+	ErrorCode string      `json:"errorCode,omitempty"`
+	ErrorText string      `json:"errorText,omitempty"`
```

Comment:
if this is just plumbing for well-known error codes, do we need an error text here? Can CNS just convert the well-known error code into some text?

Reply:
Problem is SubnetIsFull is not a well defined error code inside DNC and currently is being sent in as a text. I would like to keep both errorcode and error text for now